impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -202,6 +193,30 @@ RenderPassVK::RenderPassVK(const std::shared_ptr<const Context>& context,
   pass_info.setPClearValues(clears.data());
   pass_info.setClearValueCount(clear_count);
 
+  if (resolve_image_vk_) {
+    // If the resolve image has mip levels, only mip level 0 will be
```

Comment:
Can we not make the pass perform the transition for all levels? In fact, [I already think it does](https://github.com/flutter/engine/blob/f20283838158e3919e047d0eefdf45ea2b1c0f12/impeller/renderer/backend/vulkan/texture_source_vk.cc#L50). The only way it may not is if the image descriptor doesn't contain the right mip count.

Reply:
That value can also be [`VK_REMAINING_MIP_LEVELS`](https://registry.khronos.org/VulkanSC/specs/1.0-extensions/man/html/VkImageSubresourceRange.html#_description). But again, I suppose I don't know how we could only perform the transition for some levels.